.github/workflows/ci-isotovideo.yml

```diff
@@ -6,10 +6,14 @@ jobs:
   test:
     runs-on: ubuntu-latest
     name: "CI: Running isotovideo test"
+      #container:
+      #image: perldocker/perl-tester:5.32
     steps:
     - uses: actions/checkout@v3
     - name: Run isotovideo tests
-      run: make test-isotovideo
+      run: |
+        zypper -n in cpanm
```

Comment:
you souldn't need this, if you use the perl container above

Reply:
perl container gives `podman not found` error, which I guess is needed for the isotovideo test. So, I'm trying stuff.

But I definitely don't need this, as we're in ubuntu, not opensuse :'D